app/src/main/java/com/tristanwiley/chatse/chat/adapters/MessageAdapter.kt

```diff
@@ -94,6 +98,9 @@ class MessageAdapter(
 
     override fun getItemCount() = messages.size
 
+    override fun getItemId(position: Int): Long {
```

Comment:
when a fun consists only of a return statement, use an expression body:

`override fun getItemId(position: Int) = messages[position].messageId.toLong()`

Reply:
Thanks. will update